### PR TITLE
Bugfix: parsing of output from blkid

### PR DESCRIPTION
**Pull request by mrysanek**
Signed-off-by: Michal Rysanek <michal.rysanek@panerabread.com>

# Description
lvm will fail to determine a block device is already formatted if running under hab/effortless (which uses the busybox implementation of blkid).  busybox blkid doesn't implement any command-line switches and always returns the full block device info due to different behavior under busy……box (e.g., running in hab effortless)
Example: blkid -o value -s TYPE /dev/mapper/vg_opt-lv_logs
Expected output: 'ext4'
Actual output: /dev/mapper/vg_opt-lv_logs: UUID="5d0643bd-db86-4d23-ae02-1a574193c85a" TYPE="ext4"

## Issues Resolved
No issue submitted

## Check List
- [ X ] A summary of changes made is included in the CHANGELOG under `## Unreleased`
- [ - ] New functionality includes testing.
- [ - ] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ This file is used to list changes made in each version of the lvm cookbook.
 
 ## Unreleased
 
+## 6.1.1 - *2022-09-29*
```
Comment (Stromweld) [inline CHANGELOG.md:7]: Can you remove this? The pipeline automatically takes anything under `## Unreleased` and puts it under the version number that the pipeline determines the cookbook should be incremented to.

File: metadata.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 maintainer_email  'help@sous-chefs.org'
 license           'Apache-2.0'
 description       'Installs and manages Logical Volume Manager'
-version           '6.1.0'
+version           '6.1.1'
```
Comment (Stromweld) [inline metadata.rb:6]: Can you revert this? The pipeline will auto increment this based on a PR label.

**Review by Stromweld — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mrysanek) [reply CHANGELOG.md:7]: Will do

File: metadata.rb (hunk 2 above)
Comment (mrysanek) [reply metadata.rb:6]: Will do

**Comment by mrysanek** [comment]
Anything else I need to do?  Sorry, first time PR against sous-chefs

**Comment by Stromweld** [comment]
I'm just re-running the jobs that failed. Hashi corp's vagrant rate limit's github actions causing jobs to fail occasionally due to not being able to pull down the OS image box from vagrant.

**Review by Stromweld — APPROVED** [review]
lgtm

**Comment by kitchen-porter** [comment]
Released as: [6.1.1](https://github.com/sous-chefs/lvm/releases/tag/6.1.1)

**Pull request merged**